integreat_cms/cms/templates/users/user_form.html

```diff
@@ -80,7 +80,7 @@ <h3 class="heading font-bold text-white"><i data-feather="key" class="mr-2"></i>
 			</div>
 			<div class="px-4 pb-4 divide-y divide-gray-200 space-y-4">
                 <div class="divide-y divide-gray-200">
-                    <div>
+                    <div class="{% if not current_user.is_superuser %}hidden{% endif %}">
```

Comment:
```suggestion
                    <div class="{% if not request.user.is_superuser %}hidden{% endif %}">
```
The `request` object is available in all templates because of the [request context processor](https://docs.djangoproject.com/en/3.2/ref/templates/api/#django-template-context-processors-request)